src/main/java/ch/njol/skript/conditions/CondHasItemCooldown.java

```diff
@@ -36,15 +36,19 @@
 
 @Name("Has Item Cooldown")
 @Description("Check whether a cooldown is active on the specified material for a specific player.")
-@Examples({"if player has item cooldown on player's tool:",
-	"\tsend \"You can't use this item right now. Wait %item cooldown of player's tool for player%\""})
+@Examples({
+	"if player has player's tool on cooldown:",
+		"\tsend \"You can't use this item right now. Wait %item cooldown of player's tool for player%\""
+})
 @Since("INSERT VERSION")
 public class CondHasItemCooldown extends Condition {
 
 	static {
 		Skript.registerCondition(CondHasItemCooldown.class, 
-			"%players% (has|have) ([([an] item|a)] cooldown (on|for) %itemtypes%|%itemtypes% on cooldown)",
-			"%players% (doesn't|does not|do not|don't) have ([([an] item|a)] cooldown (on|for) %itemtypes%|%itemtypes% on cooldown)");
+			"%players% (has|have) [([an] item|a)] cooldown (on|for) %itemtypes%",
+			"%players% (has|have) %itemtypes% on cooldown",
```

Comment:
Like syntax above, this should also allow `item cooldown` (e.g. `player has {_x} on item cooldown`). Same for 4th syntax